### ISSUE TITLE
Customize archive name format

### ISSUE THREAD

**Issue by toomyem** [ctron/rpm-builder#24]
Is it possible to customize format used to create archive's name?
Currently it is created with "-arch.rpm" suffix, but there is also convention to use ".arch.rpm" (with dot before arch name).

**Comment by yarix** [comment]
the "-arch.rpm" is actually due to this bug - https://github.com/eclipse/packagedrone/issues/123

anyway, i think that it's good that you've added outputFileName (0.10.1)
it helped me workaround the issue with: `<outputFileName>${project.artifactId}-${project.version}-${rpm.release}.${rpm.architecture}.rpm</outputFileName>`


thank you.

**Comment by toomyem** [comment]
I'd like to note that I also had to manually set package name in addition to <outputFileName>, otherwise yum wasn't able to install created rpm.

```
<outputFileName>${rpm.package.name}-${project.version}-1.noarch.rpm</outputFileName>
<packageName>${rpm.package.name}</packageName>
```

**Comment by ctron** [comment]
@yarix You are right. I actually lost track of this being an issue in Package Drone. And there was no released yet which contained eclipse/packagedrone#126 … So I will keep this open and try to find a way to by quicker in making releases of the base RPM library.

**Comment by ctron** [comment]
This should be fixed in version 1.0.0 which I just released. Please re-open otherwise.
